DEPS

```diff
@@ -27,16 +27,37 @@ vars = {
   # When updating the Dart revision, ensure that all entries that are
   # dependencies of Dart are also updated to match the entries in the
   # Dart SDK's DEPS file for that revision of Dart. The DEPS file for
-  # Dart is: https://github.com/dart-lang/sdk/blob/master/DEPS
-  'dart_revision': '65a5707189dec6c5e0174cbc821acfb90f0c9b3f',
-  # Dart calls the next one "boringssl_gen_rev"
-  'dart_boringssl_gen_revision': '753224969dbe43dad29343146529727b5066c0f3',
-  # Dart calls the next one "boringssl_rev"
-  'dart_boringssl_revision': 'd519bf6be0b447fb80fbc539d4bff4479b5482a2',
-  # Dart calls the next one "observatory_pub_packages_rev"
-  'dart_observatory_packages_revision': '26aad88f1c1915d39bbcbff3cad589e2402fdcf1',
-  # Dart calls the next one "root_certificates_rev"
-  'dart_root_certificates_revision': 'a4c7c6f23a664a37bc1b6f15a819e3f2a292791a',
+  # Dart is: https://github.com/dart-lang/sdk/blob/master/DEPS.
+  # You can use //dart/tools/create_updated_fluter_deps.py to produce
```

Comment:
Something along this lines should be possible, but I thought that since the results are not guaranteed - one could get '???' instead of proper revision tag if particular dependency is dropped from dart DEPS - it's easier to leave it up to the engineer to confirm that updated dependencies look reasonable.

Also - when you say "part of the build" - what do you mean? This DEPS file is used by 'gclient sync' to get correct the sources, before the build is started.